src/app/ReadClient.cpp

```diff
@@ -113,6 +113,9 @@ ReadClient::~ReadClient()
             mpImEngine->RemoveReadClient(this);
         }
     }
+    mpExchangeMgr = nullptr;
+    mpNext        = nullptr;
+    mpImEngine    = nullptr;
```

Comment:
Why are we doing that in a destructor?

Reply:
thanks, just removed these unnecessary changes.